### PR TITLE
Lang fixes, monster eggs hopefully fixed, the list of workers is now scrollable

### LINKED ISSUE #2526: Mossy Stone Brick Monster Egg in Warehouse Level 5 build
### Minecolonies version

version 1.12.2-0.8.8618

### Expected behavior

- building Birch medieval warehouse level 5 with 1 mossy brick.

### Actual behaviour

- Upgrade template calls for 1 Mossy Stone Brick Monster Egg

### Steps to reproduce the problem

1. Build birch medieval warehouse up to level 4
2. Try to upgrade to level 5
3. Get stuck on mossy stone brick monster egg

Issue was fixed for Spruce Medieval warehouse (https://github.com/Minecolonies/minecolonies/issues/2459), but still exists in birch medieval (and potentially other level 5 medieval warehouses).

### PR DESCRIPTION
**Pull request by Nightenom**
Closes #2526

# Changes proposed in this pull request:
- Add missing strings for the guard update
- Replaced hopefully all monster eggs in medieval warehouses lvl 5
- The list of workers in the townhall is now scrollable

Review please

### CODE REVIEW

**Review by marchermans — CHANGES_REQUESTED** [review]
Build failure. Please fix the broken test.

File: src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java [hunk 1]
```diff
@@ -582,47 +580,70 @@ private void createAndSetStatistics()
...
+                    return;
+                }
+
+                final Label label = rowPane.findPaneOfTypeByID(CITIZENS_AMOUNT_LABEL, Label.class);
+                // preJobsHeaders = number of all citizens + empty row + number of all unemployed citizens
+                if (index == 0)
+                {
+                    label.setLabelText(numberOfCitizens);
+                }
+                if (index == 2)
```
[the remark this answers is not in the document]
Comment (Nightenom) [reply src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java]: It's commented above these conditions

**Comment by BuildTeamLinter** [comment]
SonarQube analysis reported 50 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 49 major

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [WindowTownHall.java#L1048](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L1048): Add a default case to this switch. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![MAJOR][MAJOR] [blocks#](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/blocks#): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [gui#](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/client/gui#): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [WindowTownHall.java#L47](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L47): class "WindowTownHall" has 41 methods, which is greater than the 35 authorized. Split it into smaller classes. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)
1. ![MAJOR][MAJOR] [WindowTownHall.java#L915](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L915): Reduce this anonymous class number of lines from 41 to at most 40, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [render#](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/client/render#): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [permissions#](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/colony/permissions#): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [requestable#](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/colony/requestable#): Add a 'package-info.java' file to document the 'requestable' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [data#](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/colony/requestsystem/data#): Add a 'package-info.java' file to document the 'data' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [init#](https://github.com/ldtteam/minecolonies/blob/cc7ef1abba811fff8185434d0d359153d4f8f874/src/main/java/com/minecolonies/coremod/colony/requestsystem/init#): Add a 'package-info.java' file to document the 'init' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/2527/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by BuildTeamLinter** [comment]
SonarQube analysis reported 50 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 49 major

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [WindowTownHall.java#L1048](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L1048): Add a default case to this switch. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![MAJOR][MAJOR] [blocks#](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/blocks#): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [gui#](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/client/gui#): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [WindowTownHall.java#L47](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L47): class "WindowTownHall" has 41 methods, which is greater than the 35 authorized. Split it into smaller classes. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)
1. ![MAJOR][MAJOR] [WindowTownHall.java#L915](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L915): Reduce this anonymous class number of lines from 41 to at most 40, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [render#](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/client/render#): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [permissions#](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/colony/permissions#): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [requestable#](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/colony/requestable#): Add a 'package-info.java' file to document the 'requestable' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [data#](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/colony/requestsystem/data#): Add a 'package-info.java' file to document the 'data' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [init#](https://github.com/ldtteam/minecolonies/blob/a4a66901d67123b1da0480f9867c77ad97d6fb39/src/main/java/com/minecolonies/coremod/colony/requestsystem/init#): Add a 'package-info.java' file to document the 'init' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**